regression/cbmc-java/annotations1/test.desc

```diff
@@ -0,0 +1,11 @@
+CORE
```

Comment:
If this test fails at this commit it should either be disabled, or this commit moved after the fix

Reply:
Or be marked as KNOWNBUG